src/solvers/flattening/boolbv.cpp

```diff
@@ -78,6 +80,15 @@ const bvt &boolbvt::convert_bv(
   return cache_entry;
 }
 
+exprt boolbvt::handle(const exprt &expr)
```

Comment:
📖

Reply:
It's already got extensive documentation in the base class.